sense/src/benchmarking.rs

```diff
@@ -11,21 +11,11 @@ benchmarks! {
 
 	create_entity {}: _(RawOrigin::Root, account("1", 0, 0), BoundedVec::truncate_from(vec![1; 256]))
 
-	mod_xp {
+	update_property {
 		let caller_origin = <T as frame_system::Config>::Origin::from(RawOrigin::Root);
+		let property_type = PropertyType::Experience;
 		Sense::<T>::create_entity(caller_origin, account("1", 0, 0), BoundedVec::truncate_from(vec![1; 1]))?;
-	}: _(RawOrigin::Root, account("1", 0, 0), 255)
-
-	mod_rep {
-		let caller_origin = <T as frame_system::Config>::Origin::from(RawOrigin::Root);
-		Sense::<T>::create_entity(caller_origin, account("1", 0, 0), BoundedVec::truncate_from(vec![1; 1]))?;
-	}: _(RawOrigin::Root, account("1", 0, 0), 255)
-
-	mod_trust {
-		let caller_origin = <T as frame_system::Config>::Origin::from(RawOrigin::Root);
-		Sense::<T>::create_entity(caller_origin, account("1", 0, 0), BoundedVec::truncate_from(vec![1; 1]))?;
-	}: _(RawOrigin::Root, account("1", 0, 0), 255)
-
+	}: _(RawOrigin::Root, account("1", 0, 0), property_type, 255)
```

Comment:
The caller should be whitelisted for DB reads/writes. Same for `create_entity` benchmark.
```
// One way
frame_benchmarking::benchmarking::add_to_whitelist;
// Another way
frame_benchmarking::whitelisted_caller;
```

Reply:
Looking into examples in the Substrate repo, it seems `RawOrigin::Root` doesn't whitelisted from the DB operations. It's not clear why, but I'm going to raise this in substrate StackExchange. For now, we can leave it as is.

Reply:
I've checked it and came to the same conclusion :)
thanks!